amazon-kinesis-client/src/test/java/software/amazon/kinesis/config/KCLReleaseCanary2XPollingH1TestConfig.java

```diff
@@ -0,0 +1,105 @@
+package software.amazon.kinesis.config;
+
+import software.amazon.awssdk.http.Protocol;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.kinesis.common.InitialPositionInStream;
+import software.amazon.kinesis.common.InitialPositionInStreamExtended;
+import software.amazon.kinesis.retrieval.RetrievalConfig;
+import software.amazon.kinesis.retrieval.polling.PollingConfig;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+public class KCLReleaseCanary2XPollingH1TestConfig implements KCLAppConfig {
+    @Override
+    public String getStreamName() {
+        return "KCLReleaseCanary2XPollingH1TestConfig";
```

Comment:
After changing the name of the class to remove 2X, I will leave the string here. We should have the "2X" in the name of the stream and the lease table in case we are testing for both KCL 1.x and 2.x using the same AWS account for clarity and to avoid resources with overlapping names between 1.x and 2.x